docs/DeveloperGuide.md

```diff
@@ -261,6 +261,17 @@ Implementation of edit booking creates a `editedBooking` before calling `Booking
 * **Alternative 2:** Verify the validity of `BookingList#setBooking` by simulating the deletion of `bookingToEdit` and addition of `editedBooking` through a method in `BookingList`
     * Pros: allows the use of `BookingList#setBooking`, separating `EditBookingCommand` from `AddingBookingCommand` and `DeleteBookingCommand`.
     * Cons: creating a method that is similar to `BookingList#contains`.
+    
+### Booking class
+
+#### Design Consideration:
+* **Alternative 1 (current choice):** Morph `Person` class into `Booking` class
+    * Pros: With the `doesOverlap` method of `Booking`, it is convenient to ensure that there are no overlapping bookings for `Residences`.
+    * Cons: All references to `Person` in existing classes, tests, and test utilities would have to be replaced, and new ones would have to be designed to accommodate `Bookings`.
+
+* **Alternative 2:** Preserve `Person` class
```

Comment:
Try to be consistent with the use of `Person` and `Persons` please! maybe `Person`s for plural?